dias/analyzers/sensitivity_analyzer.py

```diff
@@ -178,26 +163,22 @@ def run(self):
         """Task stage: analyzes data from the last period."""
         stop_time = datetime.utcnow() - self.lag
 
-        # Refresh the database
-        self.refresh_data_index()
-
-        cursor = self.data_index.cursor()
-        query = "SELECT stop FROM files ORDER BY stop DESC LIMIT 1"
-        results = list(cursor.execute(query))
-        start_time = results[0][0] if results else stop_time - self.period
 
         # Find all calibration files
-        file_list = self.find_all_archive(
-            instrument=self.instrument, data_product=self.acq_suffix
-        )
-        file_list = self.filter_files_by_time(file_list, start_time, stop_time)
+        client = Tracker("sensitivity")
+        file_list = client.new_files(filetypes=instrument)
+        start_time = stop_time - self.period
 
         if not file_list:
             err_msg = "No {}_{} files found from last {}.".format(
                 self.instrument, self.acq_suffix, self.period
             )
             raise exception.DiasDataError(err_msg)
 
+        with h5py.File(file_list[0], r) as hf:
+            start_time = ephemeris.unix_to_datetime(hf["index_map/time"][0]["ctime"])
```

Comment:
We can because I sort them by start time when I return!

It is used at some point, I need to talk to the original writers to understand if some of the parts are okay to remove.

Reply:
Ah, it is used for calculating the "inputmap", which is used for calculating baselines. So, I am going to keep that part for now.